pkgs/development/node-packages/main-programs.nix

```diff
@@ -47,6 +47,7 @@
   firebase-tools = "firebase";
   fkill-cli = "fkill";
   fleek-cli = "fleek";
+  flood = "flood";
```

Comment:
Not technically different from the `pname`, but still getting a warning so despite the comment at the top of the file, I added it.

Reply:
The comment is wrong after https://github.com/NixOS/nixpkgs/pull/246386

Reply:
At line 14:

```diff
-  # Packages that provide a single executable whose name differs from the package's `name`.
+  # Packages that provide a single or most prominent executable
```

Reply:
https://github.com/NixOS/nixpkgs/pull/247130

Reply:
This broke eval and is blocking channels:
https://hydra.nixos.org/build/230301363/nixlog/1/tail

Reply:
Simple cheap test:
```
nix eval -f. nodePackages.flood.outPath
```

Reply:
Ah. I had tested `nix build .#nodePackages.flood` and that seems to work.

Reply:
In fact, `nix eval -f. nodePackages.flood.outPath` works on my tree, so something might have broken when merging? :thinking:

Reply:
Yup, it's the interaction with #245435 :upside_down_face:.

Reply:
Fixed in #247224.